topics/product-management/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: pm
+display_name: Product Management
+short_description:  pm is a way to organize the planning, production, marketing, and tasks related to the creation, and distribution of a product.
```

Comment:
What do you think about rewording this to be something like this? 💭 

```suggestion
short_description:  Product management is the planning, production, marketing, and tasks related to the creation and distribution of a product.
```

Reply:
much cleaner thanks for suggestion